tests/src/JIT/Performance/CodeQuality/HWIntrinsic/X86/PacketTracer/PacketTracer.cs

```diff
@@ -0,0 +1,260 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using System.Runtime.Intrinsics.X86;
+using static System.Runtime.Intrinsics.X86.Avx;
+using static System.Runtime.Intrinsics.X86.Avx2;
+using static System.Runtime.Intrinsics.X86.Sse2;
+using System.Runtime.Intrinsics;
+using System;
+
+using ColorPacket256 = VectorPacket256;
+
+internal class Packet256Tracer
+{
+    public int Width { get; private set; }
+    public int Height { get; private set; }
+    private static readonly int MaxDepth = 5;
+
+    private static readonly Vector256<float> SevenToZero = SetVector256(7f, 6f, 5f, 4f, 3f, 2f, 1f, 0f);
+
+    public Packet256Tracer(int _width, int _height)
+    {
+        if ((_width % VectorPacket256.Packet256Size) != 0)
+        {
+            _width += VectorPacket256.Packet256Size - _width % VectorPacket256.Packet256Size;
+        }
+        Width = _width;
+        Height = _height;
+    }
+
+    internal unsafe void RenderVectorized(Scene scene, int* rgb)
+    {
+        Camera camera = scene.Camera;
+        // Iterate y then x in order to preserve cache locality.
+        for (int y = 0; y < Height; y++)
+        {
+            int stride = y * Width;
+            for (int x = 0; x < Width; x += VectorPacket256.Packet256Size)
+            {
+                float fx = x;
+                Vector256<float> Xs = Add(SetAllVector256(fx), SevenToZero);
+                var dirs = GetPoints(Xs, SetAllVector256<float>(y), camera);
+                var rayPacket256 = new RayPacket256(camera.Pos, dirs);
+                var SoAcolors = TraceRay(rayPacket256, scene, 0);
```

Comment:
nit: Make `0` a named parameter to help improve readability

Reply:
`0` seems already very clear as the first value of `depth` 😄

Reply:
I was indicating that, if you were to just read `TraceRay(rayPacket256, scene, 0);`, it isn't obvious that 0 is for depth (you have to go look at the `TraceRay` signature).

While `TraceRay(rayPacket256, scene, depth: 0);` makes this immediately obvious

Reply:
Oh, I see, misunderstood your words...